freshmaker/errata.py

```diff
@@ -311,22 +314,14 @@ def builds_signed(self, errata_id):
         :return: True if all builds in advisory are signed.
         :rtype: bool
         """
-        builds_per_product = self._get_attached_builds(errata_id)
-
-        # Store NVRs of all builds in advisory to nvrs set.
-        nvrs = set()
-        for builds in builds_per_product.values():
-            for build in builds:
-                nvrs.update(set(build.keys()))
-
-        # For each NVR, check that all the rpms are signed.
-        for nvr in nvrs:
-            log.info("Checking whether the build %s is signed", str(nvr))
-            build = self._errata_rest_get("build/%s" % str(nvr))
-            if "rpms_signed" not in build or not build["rpms_signed"]:
-                return False
-
-        return True
+        data_by_product = self._get_builds_by_product(errata_id)
+        log.warning(data_by_product)
```

Comment:
This log message is not so readable and includes content (product data, rpms in builds for different arches) we don't care about when there are many builds attached, I'd suggest logging a warning message with build NVR and return `False` only when a build is not signed.

Reply:
Ooops, I used it to debug some problems. Removing.